src/plugins/condition/utils/operations.js

```diff
@@ -298,12 +282,28 @@ export const OPERATIONS = [
     }
 ];
 
+function convertToNumbers(input) {
+    let numberInputs = [];
+    input.forEach(inputValue => numberInputs.push(Number(inputValue)));
+    return numberInputs;
+}
+
+function convertToStrings(input) {
+    let stringInputs = [];
+    input.forEach(inputValue => stringInputs.push(inputValue !== undefined ? inputValue.toString() : ''));
+    return stringInputs;
+}
+
+function joinValues(values, length) {
```

Comment:
same comment as above

Reply:
Would like @shefalijoshi to look at this one. joinValues was taking values and length args but length was always 1. It was then returning `values.slice(0, length).join(', ');` In all cases I could see, the first item in values was either a single value or an array, so there seemed to be no reason for the join. I just removed the function and changed all the calls to it to `values.slice(0, 1)`. I haven't seen any issues in smoke testing and all unit tests pass.

Reply:
It seems to be fine. Here's an example:
![image](https://user-images.githubusercontent.com/6541399/87352033-3e61ce80-c50f-11ea-8e89-9daab436495b.png)